CHANGELOG.txt

```diff
@@ -1,3 +1,13 @@
+2.10.0
+* Add Time functions: TimeAdd(), TimeSubtract(), and TimeDiff()
+* Add Conversion functions: ToObject(), ToArray(), ToDouble(), and ToInteger()
+* Add Logical functions: Any(), and All()
+* Add Type Check functions: IsNumber(), IsDouble(), IsInteger(), IsBoolean(), IsNull(),
+  IsBytes(), IsTimestamp(), IsDate(), IsString(), IsArray(), IsObject(), IsRef(), IsSet(),
+  IsDoc(), IsLambda(), IsCollection(), IsDatabase(), IsIndex(), IsFunction(), IsKey(),
+  IsToken(), IsCredentials(), and IsRole()
+* Make Expr constructor private
```

Comment:
did we implement this? I thought it was reverted

Reply:
It was reverted, but then we implemented it for good! You can see the discussion here: https://faunadb.atlassian.net/browse/ENG-1376